modules/c++/scene/include/scene/ProjectionModel.h

```diff
@@ -95,6 +95,18 @@ class ProjectionModel
      *  R/Rdot Contour Ground Plane Intersection
      *
      */
+    static Vector3
```

Comment:
The main use case I can think of is when doing projections prior to (or during) image formation, when the entirety of the information needed to construct a ProjectionModel is not available.